pkg/machines/helpers.es6

```diff
@@ -154,6 +154,7 @@ export function digitFilter(event, allowDots = false) {
                  event.key === 'Backspace' || event.key === 'Delete' || event.key === 'Tab' ||
                  event.key === 'ArrowLeft' || event.key === 'ArrowRight' ||
                  event.key === 'ArrowUp' || event.key === 'ArrowDown' ||
+                 (event.key === 'a' || event.ctrlKey) ||
```

Comment:
Supposedly you meant to use `&&` here instead of `||`? I don't think it needs to be overly picky here, but as you used parentheses it looks like you intended &&.

Reply:
Correct. Fixed.